### PR TITLE
scheduler: set different priorities for write-leader and write-peer

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

write-leader and write-peer need to have different priorities because write leader needs to have balanced metrics that reflect the leader, such as sum of key or qps, but write-peer does not, and write-peer does not necessarily have data on qps.

### What is changed and how it works?

set different priorities for write-leader and write-peer

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

**Comment by nolouch** [comment]
It's may conflict? especially we add qps.

**Comment by lhy1024** [comment]
> It's may conflict? especially we add qps.

In fact, the write leader's priority now has key as first priority and byte as second priority, while the write peer's priority has byte as first priority and key as second priority. They are now different.

This pr doesn't change the default behaviour, it just avoids query in the write peer's scheduling priority. because the follower doesn't have this statistic.

File: tools/pd-ctl/pdctl/command/scheduler.go [hunk 1]
```diff
@@ -539,6 +539,10 @@ func postSchedulerConfigCommandFunc(cmd *cobra.Command, schedulerName string, ar
 					schedulers.KeyPriority))
 				return
 			}
+			if priority == schedulers.QueryPriority && key == "write-peer-priorities" {
+				cmd.Println("For hotspot scheduling of write peers, it is not reasonable to use the qps dimension.")
```
Comment (Yisaer) [inline tools/pd-ctl/pdctl/command/scheduler.go:543]:
```suggestion
				cmd.Println("qps is not allowed to be set in priorities for write-peer-priorities")
```

**Review by Yisaer — APPROVED** [review]
Rest LGTM

File: tests/pdctl/scheduler/scheduler_test.go [hunk 2]
```diff
@@ -309,6 +310,15 @@ func (s *schedulerTestSuite) TestScheduler(c *C) {
 	mustExec([]string{"-u", pdAddr, "scheduler", "config", "balance-hot-region-scheduler"}, &conf1)
 	c.Assert(conf1, DeepEquals, expected1)
 
+	// write-priorities is  divided into write-leader-priorities and write-peer-priorities
```
Comment (HunDunDM) [inline tests/pdctl/scheduler/scheduler_test.go:313]:
```suggestion
	// write-priorities is divided into write-leader-priorities and write-peer-priorities
```

**Review by HunDunDM — APPROVED** [review]
rest LGTM

**Pull request merged**